app/controllers/patients_controller.rb

```diff
@@ -159,6 +161,9 @@ def create
     valid_jurisdiction = current_user.jurisdiction.subtree_ids.include?(patient.jurisdiction_id) unless patient.jurisdiction_id.nil?
     patient.jurisdiction = current_user.jurisdiction unless valid_jurisdiction
 
+    # If contact type is not set, default to "Unknown"
+    patient.contact_type = 'Unknown' if patient.contact_type.nil?
```

Comment:
Should not be nil since we set default to `Unknown`.

Reply:
This actually is needed for setting contact type to "Unknown" for imports that have that field blank